src/routes/Protected.jsx

```diff
@@ -1,13 +1,11 @@
 import React from "react";
 import { Navigate } from "react-router-dom";
 import { UserAuth } from "../context/AuthContext";
-import Dashboard from "../components/dashboard/Dashboard";
 
-export const Protected = () => {
+export const Protected = ({ children }) => {
   const { user } = UserAuth();
   if (!user) {
-    return <Navigate to="/" />;
+    return <Navigate to="/" replace />;
```

Comment:
Added replace here!. This will replace the current entry in the navigation stack instead of creating another entry on the stack